src/main/java/com/karankumar/bookproject/ui/components/toggle/PaperToggle.java

```diff
@@ -25,9 +25,9 @@
 @Tag("paper-toggle-button")
 @NpmPackage(value="@polymer/paper-toggle-button", version = "3.0.1")
 @JsModule("@polymer/paper-toggle-button/paper-toggle-button.js")
-public class SwitchToggle extends Component {
```

Comment:
We want to keep it as `SwitchToggle`, as per our master branch (see https://github.com/knjk04/book-project/tree/master/src/main/java/com/karankumar/bookproject/ui/components/toggle)

Reply:
I dont understand, even when I rebase, It's showing PaperToggle in my program